mlfinlab/online_portfolio_selection/__init__.py

```diff
@@ -13,9 +13,9 @@
 from mlfinlab.online_portfolio_selection.benchmarks.best_constant_rebalanced_portfolio import BestConstantRebalancedPortfolio
 
 # Momentum
-# from mlfinlab.online_portfolio_selection.momentum.EG import EG
-# from mlfinlab.online_portfolio_selection.momentum.FTL import FTL
-# from mlfinlab.online_portfolio_selection.momentum.FTRL import FTRL
+from mlfinlab.online_portfolio_selection.momentum.exponential_gradient import ExponentialGradient
+from mlfinlab.online_portfolio_selection.momentum.follow_the_leader import FollowTheLeader
+from mlfinlab.online_portfolio_selection.momentum.follow_the_regularized_leader import FollowTheRegularizedLeader
```

Comment:
I'm not really sure actually, do I have to expose them again? or is once enough?

Reply:
Now I think about it, most people are going to do import mlfinlab.online_portfolio_selection instead of mlfinlab.online_portfolio_selection.momentum. So, I think leaving it empty should be fine, but let me know if you think otherwise.

Reply:
We will have to test this when its merged into dev. 

Please make a note that we come back to this.